load-testing/service.socket.ts

```
@@ -0,0 +1,101 @@
+          "payload": {
+            elements: batch
+          }
+        };
+        const jsonStr = JSON.stringify(payload);
+        const buffer = new TextEncoder().encode(jsonStr).buffer;
+        this.socket.emit(SERVER_BROADCAST, roomID, buffer);
+        await setTimeout(300);
+      }
+
+      resolve();
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)